core/arch/arm/sm/sm_private.h

```diff
@@ -34,5 +34,8 @@ bool sm_from_nsec(struct sm_ctx *ctx);
 void sm_save_modes_regs(struct sm_mode_regs *regs);
 void sm_restore_modes_regs(struct sm_mode_regs *regs);
 
+/* Weak symbol for platform specific monitor service handling */
+bool sm_platform_handler(struct sm_ctx *ctx);
```

Comment:
This belongs in `<sm/sm.h>`
It could also be worth mentioning what it means if the function returns `true` or `false` as it may be a bit counter intuitive depending on how to look at it.